.travis.yml

```diff
@@ -10,19 +10,26 @@ branches:
   - trying
   - master
 
+install:
+  - sudo add-apt-repository -y ppa:ubuntu-lxc/lxd-stable;
+  - sudo apt-get -qq update;
+  - sudo apt-get -y install lxd;
```

Comment:
Could use the LXD snap instead?

Reply:
I tried that first (as the commented-out LXD_DIR environment might suggest 😄); The LXD snap doesn't seem to work in Travis.